From bfd16c2b6904336c762c562acc9b38f4352b50b0 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 23 May 2018 10:55:09 +0800
Subject: [PATCH 3864/5242] MLK-18382 ARM64: dts: freescale: imx8mq: add GPT
 compatible string

commit  10ee8683107f9eedaeb57e359d422feab22fad2e from
https://source.codeaurora.org/external/imx/linux-imx.git

i.MX8MQ uses same GPT as i.MX7D, add i.MX7D compatible
string for GPT driver, and disable it by default.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index d001ef8..26c6a3d 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -440,13 +440,14 @@
 	};
 
 	gpt1: gpt@302d0000 {
-		compatible = "fsl,imx8mq-gpt";
+		compatible = "fsl,imx8mq-gpt", "fsl,imx7d-gpt";
 		reg = <0x0 0x302d0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 55 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8MQ_CLK_GPT1_ROOT>,
 			 <&clk IMX8MQ_CLK_GPT1_ROOT>,
 			 <&clk IMX8MQ_GPT_3M_CLK>;
 		clock-names = "ipg", "per", "osc_per";
+		status = "disabled";
 	};
 
 	irqsteer_dcss: irqsteer@32e2d000 {
-- 
1.7.9.5

